src/index.js

```diff
@@ -189,21 +198,53 @@ export default class Undo {
   }
 
   /**
-   * Sets events listeners to allow keyboard actions support.
+   * Parses the keys passed in the shortcut property to accept CMD,ALT and SHIFT
+   *
+   * @param {Array} keys are the keys passed in shortcuts in config
+   * @returns {Array}
+   */
+
+  parseKeys(keys) {
+    const specialKeys = {
+      CMD: (/(Mac)/i.test(navigator.platform) ? 'metaKey' : 'ctrlKey'),
+      ALT: 'altKey',
+      SHIFT: 'shiftKey',
+    };
+    const parsedKeys = keys.slice(0, -1).map((key) => specialKeys[key]);
+    const letterKey = parsedKeys.includes('shiftKey') ? keys[keys.length - 1].toUpperCase() : keys[keys.length - 1].toLowerCase();
```

Comment:
This line might be too long.

Reply:
@camilocp98  Thanks you for the review. It was solved.